content/data-reporting/reports-api/v3.3/index.mdx

```diff
@@ -180,11 +180,12 @@ Shows how many chats occurred during the specified period.
 
 #### Response
 
-| Field               | Notes                                                    |
-| ------------------- | -------------------------------------------------------- |
-| `total`             | The total number of chats in the specified date range.   |
-| `records`           | Contains the `distribution` objects, for example, `day`. |
-| `records.day.chats` | The total number of chats that `day`.                    |
+| Field                    | Notes                                                       |
+| ------------------------ | ----------------------------------------------------------- |
+| `total`                  | The total number of chats in the specified date range.      |
+| `records`                | Contains the `distribution` objects, for example, `day`.    |
+| `records.day.total`      | The total number of chats that `day`.                       |
+| `records.day.continuous` | The number of continuous chats that `day`.                  |
```

Comment:
Is this field always returned, even if you had 0 continuous chats? If so, we should include it in the sample response (code snippet).

Reply:
This field is not always returned, we don't return 0 values in reports v3.X. I've fixed the missing response json in https://github.com/livechat/livechat-public-docs/pull/835/commits/cb7381e2ed421a62c418e786f193f35057d37b04